packages/nimble-components/src/select/index.ts

```diff
@@ -668,6 +673,17 @@ export class Select extends FormAssociatedSelect implements ErrorPattern {
         }
     }
 
+    /**
+     * @internal
+     */
+    public registerOption(option: ListOption): void {
```

Comment:
From offline discussions we will apply this same behavior to all our listbox implementations (select, combobox, and user-mentions-listbox) so they will have the same behavior and tests (with link to tech debt issue)